### ISSUE TITLE
Building photon-rs v0.2.0 fails

### ISSUE THREAD

**Issue by gp-97** [silvia-odwyer/photon#78]
### Building of photon-rs v0.2.0 fails
I am following the getting started tutorial (https://silvia-odwyer.github.io/photon/guide/getting-started/) for photon, and wrote a simple rust program to read an image.  I have no other dependency in Cargo.toml, other than photon-rs.
Building fails with following error:-
![Screenshot from 2021-03-30 01-02-21](https://user-images.githubusercontent.com/30759667/112890194-31fcc080-90f4-11eb-8aad-8a0f01222580.png)

**Comment by silvia-odwyer** [comment]
Hi @gp-97! Thanks for reporting this! I'll be releasing v0.3.0 quite soon, which will fix that issue (it's a matter of updating the `image` and `imageproc` dependencies), but until then, you could add the following to your Cargo.toml instead:

```
photon-rs = { git = "https://github.com/silvia-odwyer/photon/" } 
```

Hopefully that helps, and I'll add a comment here when v0.3.0 is released ✅

**Comment by silvia-odwyer** [comment]
I've just released Version 0.3.0 🎉 Since the dependencies are updated, it should build now, but do let me know if there are any issues! Thanks! 😄

**Comment by gp-97** [comment]
Sure. I will try again tomorrow morning

**Comment by smurrayatwork** [comment]
@silvia-odwyer Would it be possible to get newer versions of the node and web wasm files?

**Comment by silvia-odwyer** [comment]
@smurrayatwork Indeed, the updated library is now available on NPM 🎉 It's available at [photon ](https://www.npmjs.com/package/@silvia-odwyer/photon) (for the browser) and [photon-node](https://www.npmjs.com/package/@silvia-odwyer/photon-node) (for Node.js image processing). 

It's also possible to build from source, by using wasm-pack too, let me know if you want further details on that.

If you have any other questions, don't hesitate to ask too 👍

**Comment by smurrayatwork** [comment]
@silvia-odwyer The photon package from npm seems to reference `photon_rs_bg.js` that doesn't exist? 

Also, does this package require the use of wasm-loader if you don't build from source, or for the web, do you compile and instantiate yourself using the wasm file?

**Comment by silvia-odwyer** [comment]
@smurrayatwork I've just fixed it with the latest release (v0.3.1), it's now on NPM. It appears the `photon_rs_bg.js` file isn't being added by wasm-pack when building (related to [this issue here](https://github.com/rustwasm/wasm-pack/issues/199)), so I had to include it manually. It should be working now! 

If you don't build from source, wasm-loader is still a requirement yes,  but I'm hoping wasm-loader will become a part of the config files for frontend frameworks in the future, just to make loading wasm files easier. A [PR was opened in create-react-app](https://github.com/facebook/create-react-app/pull/8303), for this, hopefully it'll get merged soon!

**Comment by smurrayatwork** [comment]
@silvia-odwyer Thanks for this! I'm still having trouble getting things working with `wasm-loader` and webpack 4.x but that might just be my rules for wasm files not working or something.

**Comment by smurrayatwork** [comment]
@silvia-odwyer I'm getting errors saying that `wasm` isn't set within photon_rs_bg.js - is this related to it not doing asynchronous importing?

**Comment by gp-97** [comment]
> I've just released Version 0.3.0 tada Since the dependencies are updated, it should build now, but do let me know if there are any issues! Thanks! smile

It is building just fine. Thanks for fixing the issue and congrats for the new version. :clap:
